SDKSampleApp/Model/QRCode/QRCodeData.swift

```diff
@@ -0,0 +1,28 @@
+import Foundation
+import UIKit
+
+struct QRCodeData {
+    let urlParams: QRCodeURLParameters?
+    
+    var isContentAssetAvailable: Bool {
+        urlParams?.source != nil
+    }
+    
+    var isSourceAssetURL: Bool {
+        guard
+            let assetID = urlParams?.source
+        else {
+            return false
+        }
+        return isValidURL(assetID)
+    }
+    
+    private func isValidURL(
+        _ urlString: String
+    ) -> Bool {
+        if let url = URL(string: urlString) {
```

Comment:
Could be `guard let` instead of `if let `, because for nil you just `return false` ;)